apisix/plugins/error-log-logger.lua

```diff
@@ -134,6 +219,14 @@ local function update_filter(value)
 end
 
 
+local function send(data)
+    if config.type == "TCP" then
```

Comment:
Better to use config.tcp to decide the destination, so that we don't need to check the `config.tcp` is nil or not.